packages/p2p/src/components/advertiser-page/block-user/block-user-overlay.scss

```diff
@@ -0,0 +1,34 @@
+$header-height-offset: 23.8rem !default;
```

Comment:
Do we need `!default` here?

Reply:
Forgot to remove that T_T, ill remove it asap